### DISCUSSION TITLE
Add an option to edit vertex materials.

### DISCUSSION THREAD

**Discussion by Malik177** [ScanMountGoat/ssbh_editor#149] [Ideas]
Can you add an option to the materials editor that gives us an option to edit the vertexes. And can there also be a option to where we can modify the bones like how studio SB has?. Heres picture of studio sb
![Screenshot (553)](https://user-images.githubusercontent.com/90985970/227747339-5939840a-5d69-4d29-a6b7-29fb18c87ff0.png)

**Comment by ScanMountGoat** [comment]
> Can you add an option to the materials editor that gives us an option to edit the vertexes.

Can you clarify what you mean by vertex editing? The model's geometry can be edited in a 3D modeling application like Blender.

> And can there also be a option to where we can modify the bones

What kind of edits are you trying to make to the skeleton? If you just edit the bones, the model likely won't deform correctly during animations. I'm open to extending the Skel Editor, but I don't want to add any functionality that's better served by editing the model and exporting it again from Blender, 3DS Max, etc.

**Comment by Malik177** [comment]
oh alright. also what blender addon can edit the geometry of these models? also is there one that can import them in and then export them out in the same format as they were before?

**Reply by ScanMountGoat** [reply]
It's linked on the bottom of the ssbh editor README. You can import and export directly the smash formats like numshdb, numdlb, etc. https://github.com/ssbucarlos/smash-ultimate-blender
